### DISCUSSION TITLE
Wolfi Community Call Next Wednesday 8/2!

### DISCUSSION THREAD

**Discussion by jdolitsky** [wolfi-dev/os#3876] [Announcements]
📅 Wed, 8/2 @ 12pm ET
🐙 K3s on Wolfi in 3 different ways
💐 Q&A!
Add invite to your calendar:
https://calendar.google.com/calendar/u/0/r/eventedit/copy/aWpydnZxc2VhaHBlMzY5OWIzN2xvdGJmYXZfMjAyMzA4MDJUMTYwMDAwWiBjXzdlYzYwZjQ4NTkzMWY5MDU2MDQwYTNlMjQyNzM0MDBkZTQxYTE0M2VjNjA3MDNiNDExZDc3YjFmNTM0ZWMxNWZAZw

**Comment by kapilt** [comment]
any meeting notes published on these meetings?
